evm/hyperliquid.js

```diff
@@ -334,22 +305,43 @@ async function main(options) {
 if (require.main === module) {
     const program = new Command();
 
-    program.name('HyperliquidBlockHelper').description('Script to manage Hyperliquid specific actions');
+    program.name('hyperliquid').description('Hyperliquid chain management commands');
+
+    // Update block size command
+    const updateBlockSizeCmd = program
+        .command('update-block-size')
+        .description('Update Hyperliquid block size')
+        .addOption(new Option('--block-size <blockSize>', 'block size to switch to').choices(['big', 'small']).makeOptionMandatory(true));
+
+    addEvmOptions(updateBlockSizeCmd, { privateKey: true });
+
+    updateBlockSizeCmd.action((options) => {
+        options.action = 'updateBlockSize';
```

Comment:
see `stellar/its.js` for how to do this cleanly without needing such workarounds